### PR TITLE
:art: (docs & ci) rework scripts and templates

### PR DESCRIPTION
**Pull request by hairmare**
* [branch root](https://github.com/hairmare/helm-charts/tree/style/manage-helm-docs-tpl)
* [x] add badges to root readme
* [x] refactor ci scripts into hack/ci folder
* [x] consistent naming in ci scripts
* [x] combine tst and pre-commit ci config
* [x] improve auto install for ci tools
* [x] refactor chart readme with some 0.14 features ([template](https://github.com/hairmare/helm-charts/blob/style/manage-helm-docs-tpl/hack/config/helm-docs/README.md.gotmpl))
* [x] add sales blurb to individual chart readmes
* [x] basic documentation on templates and dev tools
* [x] configure codeowners
* [x] configure issue/pr templates
* [x] add contributing
* [x] configure gitignore
* [x] simple makefile for make users
* [x] update domain to adfinis.com
* [x] consistent maintainers setup (w/o personal accounts)
* [x] add sources where missing
* [x] add adfinis repo to ct.yaml

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+**Affected chart**
+Tell us which chart this bug is about.
```
Comment (tongpu) [inline .github/ISSUE_TEMPLATE/bug_report.md:10]:
```suggestion
Tell us which chart is affected by the bug.
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Logs**
+If applicable, add logs or screenshots to help explain your problem.
```
Comment (tongpu) [inline .github/ISSUE_TEMPLATE/bug_report.md:26]:
```suggestion
If applicable, add logs or screenshots to help illustrate the problem.
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+**Affected version**
+Which version of the chart were you using.
+
+**Values**
+Let us know of any values you are setting.
```
Comment (tongpu) [inline .github/ISSUE_TEMPLATE/bug_report.md:32]:
```suggestion
Let us know of any specific values you have configured.
```

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
+---
+name: Feature request
+about: Suggest an idea for a chart
+title: ''
+labels: ''
+assignees: ''
+
+---
+**Is your feature request related to a problem? Please describe.**
+A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
```
Comment (tongpu) [inline .github/ISSUE_TEMPLATE/feature_request.md:10]:
```suggestion
A clear and concise description of what the problem is, e.g. "I'm always frustrated when [...]"
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+If applicable, add logs or screenshots to help illustrate the problem.
+
+**Affected version**
+Which version of the chart were you using.
+
+**Values**
+Let us know of any specific values you have configured.
+
+**Environment**
+What version of Kubernetes from what vendor are you using.
```
Comment (eyenx) [inline .github/ISSUE_TEMPLATE/bug_report.md:35]: shouldn't we also suggest adding the helm version? Perhaps even adding example code to get the output we need. 

i.e. `kubectl version` and `helm version`

File: .github/workflows/release.yaml [hunk 6]
```diff
@@ -9,14 +9,12 @@ jobs:
...
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
 
-      - name: Fetch history
-        run: git fetch --prune --unshallow
-
-      - name: Install deps
-        run: .ci/github/install-release-deps.sh
+      - name: Run hack/ci/github/setup/release.sh
```
Comment (eyenx) [inline .github/workflows/release.yaml:16]: I would prefer setting the name to what te script actually does.

File: .github/workflows/release.yaml [hunk 7]
```diff
@@ -28,7 +26,7 @@ jobs:
         env:
           CR_TOKEN: "${{ secrets.CR_TOKEN }}"
 
-      - name: Update Pages site
-        run: .ci/github/update-pages-site.sh
+      - name: Run hack/ci/github/steps/update-page.sh
```
Comment (eyenx) [inline .github/workflows/release.yaml:29]: Here too, what doesn this script do?

File: CONTRIBUTING.md [hunk 8]
```diff
@@ -0,0 +1,6 @@
+# Contributing
+
+Contributions to the Adfinis Helm Charts are very welcome! Have a look at the open [issues](https://github.com/adfinis-sygroup/helm-charts/issues)
+and open a [GitHub pull request](https://github.com/adfinis-sygroup/helm-charts/compare). See our [docs](./docs/development.md) for how to setup a
```
Comment (eyenx) [inline CONTRIBUTING.md:4]: we are in fact on GitHub, do we need to tell people to open a "GitHub pull request"? IMHO "open a pull request" would suffice.

File: README.md [hunk 9]
```diff
@@ -19,36 +19,41 @@ helm repo add adfinis https://charts.adfinis.com
...
-pre-commit install-hooks
+# update charts README
+hack/helm-docs.sh
 ```
 
+We track issues with this chart repository in the [issue tracker](https://github.com/adfinis-sygroup/helm-charts/issues).
+
 ## About this repository
 
 Adfinis fights for a software world that is more open, where the quality is
```
Comment (eyenx) [inline README.md:59]: where the quality is what?

File: .github/workflows/release.yaml (hunk 6 above)
Comment (hairmare) [reply .github/workflows/release.yaml:16]: It sets up the environment needed for the release.yaml github workflow.

File: .github/workflows/release.yaml (hunk 7 above)
Comment (hairmare) [reply .github/workflows/release.yaml:29]: It runs the update-page step on our github ci solution. Isn't that all in the path/filename?

**Comment by eyenx** [comment]
I would welcome a better naming then "Run <script>" 


-- 

We are now called Adfinis! More at www.adfinis.com

Adfinis AG
Antonio Tauro, System Engineer, GPG KeyID: 0x0796132F0077A5F8
Güterstrasse 86 | CH-4053 Basel
Office +41 61 500 31 31 | Direct +41 61 500 31 37
www.adfinis.com

**Comment by hairmare** [comment]
> I would welcome a better naming then "Run <script>"

OK, got it. The "Run ..." naming was an attempt to make everything look consistent in the ui. Looking back it doesn't seem to make sense.

**Comment by eyenx** [comment]
I just find it's redundant to have 

```
name: run <script_path>
run: <script_path>
```


-- 

We are now called Adfinis! More at www.adfinis.com

Adfinis AG
Antonio Tauro, System Engineer, GPG KeyID: 0x0796132F0077A5F8
Güterstrasse 86 | CH-4053 Basel
Office +41 61 500 31 31 | Direct +41 61 500 31 37
www.adfinis.com

**Pull request merged**